client/app/components/project/project-editor-list-item.js

```diff
@@ -0,0 +1,34 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { optionset } from '../../helpers/optionset';
+
+export default class ProjectEditorListComponent extends Component {
+  get disableDelete() {
+    return this.args.disableDelete ?? false;
+  }
+
+  @tracked removeEditorModalOpen = false;
+
+  get isPrimaryApplicantOrContact() {
+    const { role } = this.args;
+    return role === optionset(['projectApplicant', 'applicantrole', 'code', 'PRIMARY_CONTACT'])
+      || role === optionset(['projectApplicant', 'applicantrole', 'code', 'PRIMARY_APPLICANT']);
+  }
+
+  @action
+  tryRemoveEditor() {
+    this.removeEditorModalOpen = true;
+  }
```

Comment:
okay so this action just opens the initial modal before the user confirms? "try" feels a bit confusing to me, not sure what would be a better name though, maybe `openRemoveEditorModal` or something?

Reply:
@trbmcginnis  Yeah, I also had a hard time naming this. `openRemoveEditorModal` sounds good!